mne/tests/test_source_estimate.py

```diff
@@ -679,6 +679,150 @@ def test_center_of_mass():
     assert_equal(np.round(t, 2), 0.12)
 
 
+@testing.requires_testing_data
+@pytest.mark.parametrize(
+    "label_type, mri_res, test_label, cf, call",
+    [
+        (str, False, False, "head", "meth"),  # head frame
+        (str, False, str, "mri", "func"),  # fastest, default for testing
+        (str, True, str, "mri", "func"),  # fastest, default for testing
+        (str, True, False, "mri", "func"),  # mri_resolution
+        (list, True, False, "mri", "func"),  # volume label as list
+        (dict, True, False, "mri", "func"),  # volume label as dict
+    ],
+)
+def test_extract_label_time_course_volume_pca_flip(
+    src_volume_labels, label_type, mri_res, test_label, cf, call
```

Comment:
We always want tests to be fairly complete, but competing against that we also want them to be as quick as possible. Our test suite takes over an hour to run already! 😢 

Do we need to iterate over `cf` here for example? If we already test that functionality elsewhere and the `pca_flip` functionality here is independent of that, let's not. Same thing with `mri_resolution`, this seems like something that is unrelated and tested elsewhere most likely.

Maybe the `pca_flip` option could just be added somewhere else (maybe as one parameter in an already parametrized function) to really trim it down?

Reply:
In this case I think you're over-testing quite a bit. Your code change is only relevant to some minor parts of the code, not coordinate frame, mri resolution, etc. So I think it makes more sense to trim it down (a lot!) or integrate elsewhere